quickstart/kubernetes.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Kubernetes
+---
+
+To learn how to create Kubernetes applications using Pulumi, please explore any of these tutorials:
+
+* [Create a Guestbook with Redis and PHP](./kubernetes-guestbook.html): Build a simple multi-tier
+  application -- a guestbook -- using Kubernetes
```

Comment:
Do we need this layer of indirection?  The other top level topics just have one example each directly linked from the TOC, and finish with links off to other "next steps" for that area in other places (examples, etc.).  Should we do the same here for now?

Reply:
I plan to refactor all of these into AWS, Azure, Google Cloud, Kubernetes, and Cloud Framework, each with its own landing page of multiple tutorials, as part of https://github.com/pulumi/docs/issues/500. This is in line with that. If you hate it, we can consider alternative approaches, but this seems like a pretty intuitive way to go about it.